src/wp-admin/includes/bookmark.php

```diff
@@ -126,6 +126,9 @@ function wp_delete_link( $link_id ) {
  */
 function wp_get_link_cats( $link_id = 0 ) {
 	$cats = wp_get_object_terms( $link_id, 'link_category', array( 'fields' => 'ids' ) );
+	if ( is_wp_error( $cats ) ) {
+		return array();
+	}
 	return array_unique( $cats );
```

Comment:
Code review discussion with @jrfnl:

- Use the same strategy as in PR 736 by flipping this around to check that `wp_get_object_terms` does return an array and then pass that into `array_unique()`; else, return an empty `array()`.
- We do not need to return the `WP_Error`. Why? This function expects to return only an array.

```php
if ( is_array( $cats ) {
	return array_unique( $cats );
}

return array();
```